Client/src/components/common/Table/index.js

```diff
@@ -2,7 +2,8 @@ import React from 'react';
 import Cell from './cell';
 import './index.css';
 
-const Table = ({ headings, rows, history }) => {
+const Table = ({ headings, rows, history, pathname }) => {
```

Comment:
@Fatmasiam If you want, you could try using `withRouter()` in this component to get history and pathname:
https://github.com/ReactTraining/react-router/blob/master/packages/react-router/docs/api/withRouter.md

It's a HOC, so you wrap your component with it when you export it:
`export default withRouter( table );`

And then it injects history, location (which you can use to get pathname) and match into your component's props for you, magically, so that you don't have to pass them to the component yourself.
So then every time you use a `<Table>`, you never again have to pass it `history` / `pathname` yourself.  It makes things easier.